explore/src/main/scala/explore/observationtree/ObsSummaryTable.scala

```diff
@@ -149,108 +151,111 @@ object ObsSummaryTable extends TableHooks:
-                  ^.onClick ==> (_.stopPropagationCB *> Callback(
-                    cell.row.getToggleExpandedHandler()()
-                  ))
-                )(Icons.ChevronRightLight.withFixedWidth(true))
-              else "",
-            size = 18.toPx,
-            enableResizing = false
-          ),
-          column(TargetTypeColumnId, _ => ())
-            .setCell(_ => Icons.Star.withFixedWidth())
-            .setSize(35.toPx),
-          column(TargetColumnId, identity, identity).setCell { cell =>
-            def targetUrl(obsId: Observation.Id, tWId: TargetWithId) = <.a(
-              ^.href := ctx.pageUrl(AppTab.Targets, props.programId, Focused.target(tWId.id)),
-              ^.onClick ==> (e =>
-                e.preventDefaultCB *> e.stopPropagationCB *> ctx.pushPage(
-                  AppTab.Observations,
-                  props.programId,
-                  Focused.singleObs(obsId, tWId.id.some)
-                )
-              ),
-              tWId.target.name.value
-            )
+      def targetUrl(obsId: Observation.Id, tWId: TargetWithId) = <.a(
+        ^.href := ctx.pageUrl(AppTab.Observations,
+                              props.programId,
+                              Focused.singleObs(obsId, tWId.id.some)
+        ),
+        ^.onClick ==> (e =>
+          e.preventDefaultCB *> e.stopPropagationCB *> ctx.pushPage(
+            AppTab.Observations,
+            props.programId,
+            Focused.singleObs(obsId, tWId.id.some)
+          )
+        ),
+        tWId.target.name.value
+      )
 
-            cell.value match {
-              case ExpandedTargetRow(obsId, target) => targetUrl(obsId, target)
-              case ObsRow(obs, _, _)                => obs.title
-            }
-          },
-          column(ConstraintsColumnId, r => (r.obs.id, r.obs.constraintsSummary)).setCell(cell =>
+      List(
+        // TODO: GroupsColumnId
+        ColDef(
+          ColumnId("expander"),
+          cell = cell =>
+            if (cell.row.getCanExpand())
+              <.span(
+                ^.cursor.pointer,
+                ExploreStyles.ExpanderChevron,
+                ExploreStyles.ExpanderChevronOpen.when(cell.row.getIsExpanded()),
+                ^.onClick ==> (_.stopPropagationCB *> Callback(
+                  cell.row.getToggleExpandedHandler()()
+                ))
+              )(Icons.ChevronRightLight.withFixedWidth(true))
+            else "",
+          enableResizing = false
+        ).setSize(35.toPx),
+        column(ObservationIdColumnId, _.obs.id),
+        // TODO: ValidationCheckColumnId
+        column(StatusColumnId, _.obs.status),
+        // TODO: CompletionColumnId
+        // TODO: TargetTypeColumnId
+        column(TargetTypeColumnId, _ => ())
+          .setCell(_ => Icons.Star.withFixedWidth())
+          .setSize(35.toPx),
+        column[String | TagOf[Anchor]](TargetColumnId, _.obs.title, r => targetUrl(r.obsId, r.targetWithId)),
```

Comment:
Is that the return type of `targetUrl`?

Reply:
`targetUrl` is `TagOf[Anchor]`, `obs.title` is `String`